src/actions/AddSiteActions.js

```diff
@@ -82,17 +85,31 @@ export const updateAccessibilityOption = ({accessibilityOption}) => {
     }
 };
 
-export const updateFacilitiesOption = ({facilitiesOption}) => {
+export const updatePriceOption = ({priceOption}) => {
     return {
-        type: SITE_FACILITIES_OPTION_CHANGED,
-        payload: {facilitiesOption}
+        type: SITE_PRICE_OPTION_CHANGED,
+        payload: {priceOption}
     }
 };
 
-export const updatePriceOption = ({priceOption}) => {
+export const updateCountyOption = ({countyOption}) => {
     return {
-        type: SITE_PRICE_OPTION_CHANGED,
-        payload: {priceOption}
+        type: SITE_COUNTY_OPTION_CHANGED,
+        payload: {countyOption}
+    }
+};
+
+export const updateForestOption = ({forestOption}) => {
+    return {
+        type: SITE_FOREST_OPTION_CHANGED,
+        payload: {forestOption}
+    }
+};
+
+export const updateMVUMOption = ({mvumOption}) => {
+    return {
+        type: SITE_MVUM_OPTION_CHANGED,
+        payload: {mvumOption}
```

Comment:
GH-93 -- Above are the actions for county, forest, and MVUM which an admin can now add directly when uploading a site.